pkg/webhook/webhook.go

```diff
@@ -19,25 +19,24 @@ package webhook
 import (
 	"fmt"
 
+	"github.com/k8snetworkplumbingwg/kubemacpool/pkg/names"
+	"github.com/k8snetworkplumbingwg/kubemacpool/pkg/pool-manager"
+	webhookserver "github.com/qinqon/kube-admission-webhook/pkg/webhook/server"
+	"github.com/qinqon/kube-admission-webhook/pkg/webhook/server/certificate"
 	admissionregistration "k8s.io/api/admissionregistration/v1beta1"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/client-go/kubernetes"
-
 	"sigs.k8s.io/controller-runtime/pkg/manager"
-	runtimewebhook "sigs.k8s.io/controller-runtime/pkg/webhook"
-	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
 
-	"github.com/k8snetworkplumbingwg/kubemacpool/pkg/names"
-	"github.com/k8snetworkplumbingwg/kubemacpool/pkg/pool-manager"
+const (
+	mutatingWebhookConfigurationName = "kubemacpool-mutator"
```

Comment:
We don't need this to be a constant. Neither is the port number

Reply:
agree regarding the constant.
Regarding the port number - [webhookserver.New()](https://github.com/qinqon/kube-admission-webhook/blob/48024d9331d572a8335862974211784b76e7ad24/pkg/webhook/server/server.go#L32) function uses default port of 8443.
Shouldn't it be the same as the port of the service, described in [CreateOwnerRefForService](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/599529e12198a7c3e0f7aef5bdb60fc514ad045b/pkg/webhook/webhook.go#L149)()? (this is the reason I changed the port to 8000 here)

Reply:
Good that their match I think. Could you use a constant for this since it is shared across multiple places?